### ISSUE TITLE
The rule doesn't always work

### ISSUE THREAD

**Issue by ABTOP55** [uBlock-LLC/uBlock#1297]
I created a rule for blocking. But it works only once - right after its creating. Then the object from the rule isn't blocked on the webpages.
Here is an expamle, pages which the rule doesn't work on 
http://rutor.org/torrent/411258/igra-prestolov-odin-den-iz-zhizni_game-of-thrones-a-day-in-the-life-2015-hdtvrip-720p-kravec/fox

<img src="http://i.imgur.com/jfBMA2Z.png][img]http://i.imgur.com/rqU708G.png">
<img src="http://i.imgur.com/QDlsJMV.png][img]http://i.imgur.com/H6RDmmN.png">

**Comment by lewisje** [comment]
It's better to type out the rules...

`||changecopyright.ru/bannersout/*` should work well, although the ending asterisk is not needed (it is only needed if the pattern otherwise starts and ends with a forward slash, to keep it from being interpreted as a regular expression).

`||http://changecopyright.ru/bannersout/banner-468x60.png` is not what you want: The `||` means "the following domain and all subdomains" and it can only appear right before a domain name (so this literally means "match URLs from the domain `http` and all subdomains, followed by a colon with no port number..."; you may have wanted the single `|` because that means "the URL must begin/end here" and within a pattern, it can only appear at the start or the end. However, this rule, if fixed, would be made redundant by your first rule.

Also, just post the bare URLs for the images, not the BBCode: That's not how images are embedded in Markdown, which is what GitHub uses, and anyway, it's better to post image links than to embed images in GitHub issues.

**Comment by sn260591** [comment]
This banner is in RU AdList exceptions. 
Why the user filters have a lower priority, than subscriptions?

**Comment by lewisje** [comment]
Oh, I didn't notice that; fortunately, uBlock has just the kind of syntax extension you're looking for: https://github.com/chrisaljoudi/uBlock/wiki/Filter-syntax-extensions#network-filters

`||changecopyright.ru/bannersout/$important` will override whitelists in any subscription; `$important` is not available in whitelist rules or in subscriptions.

If you want to override even _that_ you will need to use dynamic filtering, which is similar to a set of uMatrix rules and is much less granular.

**Comment by ABTOP55** [comment]
<b>lewisje</b>, Thank you. <i>||changecopyright.ru/bannersout/$important</i> - what I need
<b>sn260591</b>, subscription RU AdList not use. I only own rules
